www/js/app.js

```diff
@@ -3336,4 +3337,19 @@ function goToMainArticle () {
     });
 }
 
+function handlePrintButton() {
```

Comment:
You don't need to add these event listeners inside a function. Just add them straight after the spot in app.js where clicking the random button is defined. Remove the handlePrintButton() wrapper function.